csvGenerator.ipynb

```diff
@@ -0,0 +1,195 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "6f275807",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import csv\n",
+    "import math\n",
+    "import random\n",
+    "import requests\n",
+    "from CitesphereConnectorTest import *\n",
+    "from CitesphereConnector import CitesphereConnector"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "2c0597b1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "auth_object = authObject()\n",
+    "auth_object.authType  = 'oauth'\n",
+    "auth_object.access_token=\"access-token\"\n",
+    "connector = CitesphereConnector(\"https://diging-dev.asu.edu/citesphere-review/api\", auth_object)\n",
```

Comment:
the url string should be a place holder (the actual url should not be pushed to github) and the string needs to be a constant defined at the beginning.